module/observationSubmissions/validator/v1.js

```diff
@@ -26,6 +26,15 @@ module.exports = (req) => {
 
         pushIncompleteObservationSubmissionForReporting: function () {
             req.checkParams('_id').exists().withMessage("required submission id")
+        },
+
+        list: function () {
+            req.checkParams('_id').exists().withMessage("required observation id")
+            .isMongoId().withMessage("Invalid observation id");
+            req.checkQuery('entityId').exists().withMessage("required entity id")
+            .isMongoId().withMessage("Invalid entity id");
+            req.checkQuery('solutionId').exists().withMessage("required solution id")
```

Comment:
@aman-tunerlabs - Why do we need solution ID for listing observation submission? Isn't observation ID and entity ID enough ?